cypress/integration/welcoming-integration/Welcoming.spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+import { expect } from 'chai';
+describe('Welcoming flow', () => {
+    beforeEach(() => {
+        cy.visit('/');
+        cy.request('GET', '/helloworld.json').should((response) => {
+            expect(response.status).to.eq(200);
+        });
+    });
+
+    it('Check navigating', () => {
+        cy.document().should('have.property', 'charset').and('eq', 'UTF-8');
+        cy.title().should('include', 'React App template');
+
+        cy.get('#helloWorldId').should('be.visible');
```

Comment:
Maybe used data-test-id like below :
`cy.get('[data-test-id="test-example"]').should('have.class', 'example')`

cf. https://example.cypress.io/commands/querying

Modification needed in HelloWorld component too if the comment retained

Reply:
done